docs/decisions/0009-versioning-for-phdi-lib.md

```diff
@@ -0,0 +1,68 @@
+# 9. Formalize software version numbers using Semantic Versioning (SemVer)
```

Comment:
Can we shorten this to something like "Implement Semantic Versioning"? At the very least, I don't think the shortening of "Semantic Versioning" should be put in the title of the ADR.